src/diffpy/pdfgui/control/pdfguimacros.py

```diff
@@ -381,7 +381,7 @@ def makeDopingSeries(control, fit, base, dopant, paths, doping):
     from diffpy.pdfgui.control.pdfguicontrol import PDFGuiControl
 
     control = PDFGuiControl()
-    control.load("../../tests/testdata/ni.ddp")
+    control.load("src/diffpy/pdfgui/tests/testdata/ni.ddp")
```

Comment:
did you test this?  It is unlikely to work because this is a relative file-path.